---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* B.S. in Biomathematics and Mathematics, Rutgers University, 2016
* M.S. in Biomathematics, University of California, Los Angeles, 2018
* Ph.D in Biomathematics, University of California, Los Angeles, 2021 (expected)

Research experience
======
* October 2012 – Summer 2013: Experiment Assistant
  * CABM Structural Bioinformatics Laboratory (@ Rutgers)
  * Duties included: His-tag Chemical Binding Chromatography, using “Cap” method finding protein crystal, primer designing, DNA cloning, and X-ray Crystallography.
  * Supervisor: Professor Guy Montelione

* Summer 2013: Undergraduate Research Fellow
  * Summer Undergraduate Research Experience (SURE) Program & Byrne Seminar Research Program in CABM (Rutgers)
  * Duties included: Focus on solving protein (Enzyme+Coenzyme+Cofactor) structure by crystallization (focus on Crystal structure with enzyme and cofactor), using maltose-binding-protein to increase the solubility of enzyme, Zygosaccharomyces rouxii. ATCC 2623
  * Supervisor: Professor Guy Montelione

* Summer 2014: Bernstein Scholarship Research Fellow
  * Bernstein Scholarship Summer research in the Center for Integrative Proteomics (Montelione Lab)
  * Duties included: Studying and comparing the similarity as well as difference of disorder-region within proteins between protein crystal structure data (through experiments) and software prediction (such as SEG) by using data mining to analyze data from Protein Data Bank (PDB).
  * Supervisor: Professor Guy Montelione
  
* Summer 2014 – November 2015: Team member, Wetlab core member
  * BIOMOD competition, Rutgers Team “Nzymes ‘R’ Us”, Khare Lab, Proteomics, Piscataway, NJ
  * Duties included: (1) Team Project Website: http://eden.rutgers.edu/~biomod/index.html; (2) Design Primer/Gblock for target PDZ/SH3 domains; (3) Perform Site directed mutagenesis, Amplification, and Gibson Assembly to insert target DNA (DNA for PDZ/SH3 domains) into plamid (pet16 in our case); (4) Express and purify PDZ/SH3 domain proteins; (5) Analyze fluorescent protein mutation impact by using Fluorescent Assay; (6) Completing the Wet lab part of the Project Website (Including Lab protocol, Equipment, and Experimental Status): http://eden.rutgers.edu/~biomod/experiments.html.
  * Supervisor: Professor Sagar Khare
  
* June 2015 – July 2016: Independent interdisciplinary undergraduate researcher
  * Honors Interdisciplinary Undergraduate Thesis: Predicting and Enhancing Enzyme Clustering Efficiency, collaboration with Khare Lab & Prof. Eduardo Sontag, Proteomics.
  * Duties included: (1) Analyze preliminary research in enzyme scaffolding and enzyme clustering; (2)Create toy model to simulate enzymatic reaction within a cell and change to distribution of enzyme in order to find the optimum enzyme distribution.
  * Supervisor: Professor Eduardo Sontag & Sagar Khare

Skills
======
* Intermediate Skill in X-Ray Crystallography
* Biochemistry and Genetics Lab Operation and Regulation
* Programming:
  * Matlab
  * R
  * Julia
  * Python
  * Java

Publications
======
- Yang, Lu, Elliott M. Dolan, Sophia K. Tan, **Tianyun Lin**, Eduardo D. Sontag, and Sagar D. Khare. Computation‐Guided Design of a Stimulus‐Responsive Multienzyme Supramolecular Assembly. *ChemBioChem* 18, no. 20 (2017): 2000-2006. [LINK](https://onlinelibrary.wiley.com/doi/full/10.1002/cbic.201700425)

## In Preparation/Review

- Samraat Pawar, Anthony I Dell, **Tianyun Lin**, Daniel J Wieczynski and Van M Savage. Interaction dimensionality scales up to generate bimodal consumer-resource size-ratio distributions in ecological communities. (In review)
  
Talks
======
  <ul>{% for post in site.talks %}
    {% include archive-single-talk-cv.html %}
  {% endfor %}</ul>
  
Teaching
======
  <ul>{% for post in site.teaching %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Service and leadership
======
* Currently signed in to 43 different slack teams
